stripe/src/main/java/com/stripe/android/model/StripeJsonUtils.kt

```diff
@@ -0,0 +1,248 @@
+package com.stripe.android.model
+
+import androidx.annotation.Size
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * A set of JSON parsing utility functions.
+ */
+internal object StripeJsonUtils {
+    private const val EMPTY = ""
+    private const val NULL = "null"
+
+    /**
+     * Calls through to [JSONObject.optInt] only in the case that the
+     * key exists. This returns `null` if the key is not in the object.
+     *
+     * @param jsonObject the input object
+     * @param fieldName the required field name
+     * @return the value stored in the requested field, or `null` if the key is not present
+     */
+    @JvmStatic
+    fun optBoolean(
+        jsonObject: JSONObject,
+        @Size(min = 1) fieldName: String
+    ): Boolean? {
+        return if (!jsonObject.has(fieldName)) {
+            null
+        } else jsonObject.optBoolean(fieldName)
```

Comment:
how does ktlint allow braces for the if w/o braces in the else
my eyes 😭

Reply:
😢